### ISSUE TITLE
Shorten uses of module names in sentences

### ISSUE THREAD

**Issue by chasenlehara** [canjs/canjs#2866]
It’s more difficult to read paragraphs that include module names like `can-connect/cache-requests/cache-requests`.

Where we can, I think it would be good to either:
1. Use a textual description of the API (e.g. “Cache Requests behavior”)
2. Shorten the module name (e.g. “can-connect/cache-requests/”)

**Comment by gKreator** [comment]
Maybe use a title tooltip on the module name to display the textual description. Or vice versa. 

The shorthand module name is as far as I know a StealJs thing. Therefore it can be confusing to people using other module loaders. I would be careful in this and state this clearly. I would see there being a toggle button to switch between the views.

**Comment by chasenlehara** [comment]
Yeah, maybe abbreviating the module name (StealJS style) is a bad idea. If this replacement is just in sentences, not the API examples, do you think it’d still be confusing to non-Steal users?

[Here’s](http://canjs.com/doc/guides/chat.html#CreateMessages) the kind of thing I want to fix:

> This is because [can-connect/can/super-map/super-map](http://canjs.com/doc/can-connect/can/super-map/super-map.html) adds the [can-connect/real-time/real-time](http://canjs.com/doc/can-connect/real-time/real-time.html) behavior. The [can-connect/real-time/real-time](http://canjs.com/doc/can-connect/real-time/real-time.html) behavior automatically inserts newly created messages into lists that they belong within.

Both uses of `can-connect/real-time/real-time` could be replaced with “[real-time](http://canjs.com/doc/can-connect/real-time/real-time.html)”, like so:

> This is because [can-connect/can/super-map/super-map](http://canjs.com/doc/can-connect/can/super-map/super-map.html) adds the [real-time behavior](http://canjs.com/doc/can-connect/real-time/real-time.html). The [real-time behavior](http://canjs.com/doc/can-connect/real-time/real-time.html) automatically inserts newly created messages into lists that they belong within.

`can-connect/can/super-map/super-map` is still a mouthful, which is why I thought `can-connect/can/super-map/` might be better:

> This is because [can-connect/can/super-map/](http://canjs.com/doc/can-connect/can/super-map/super-map.html) adds the [real-time behavior](http://canjs.com/doc/can-connect/real-time/real-time.html). The [real-time behavior](http://canjs.com/doc/can-connect/real-time/real-time.html) automatically inserts newly created messages into lists that they belong within.

Thoughts on that?

**Comment by marshallswain** [comment]
I would just shorten it to the file name, so it would say 'real-time' and those who want the full module name can click on it to view its docs

**Comment by chasenlehara** [comment]
[Here’s](http://localhost/GitHub/canjs/doc/can-core.html#can_define) another sentence that could benefit from using Steal-style abbreviations:

> [can-define/map/map](http://localhost/GitHub/canjs/doc/can-define/map/map.html) and [can-define/list/list](http://localhost/GitHub/canjs/doc/can-define/list/list.html) allow you to create observable maps and lists with well defined properties.

This would be easier to read as:

> [can-define/map/](http://localhost/GitHub/canjs/doc/can-define/map/map.html) and [can-define/list/](http://localhost/GitHub/canjs/doc/can-define/list/list.html) allow you to create observable maps and lists with well-defined properties.

**Comment by phillipskevin** [comment]
I think in lots of cases it would be better to abbreviate using the most common variable name for the module.

For example, in the docs below, the link to [can-connect/constructor/constructor](http://canjs.com/doc/can-connect/constructor/constructor.html) could just say `constructor` since that is what we use whenever writing `var constructor = require("can-connect/constructor/constructor");`.

![image](https://cloud.githubusercontent.com/assets/5851984/21119755/97845124-c089-11e6-97d1-8da225caea22.png)

In most cases, we have a common variable naming convention like this that we can use (like `DefineMap` for `can-define/map/map`)

**Comment by gKreator** [comment]
Yes, I agree. Using the common variable name is allot smoother. I would just have the real name as a tool tip at least.
